### PR TITLE
Fixed the connector for FluxFM web radio.

### PR DESCRIPTION
**Pull request by fckawe**
**Describe the changes you made**
The connector didn't work anymore after the website was redesigned.
Album info and track art aren't available anymore. Artist and title cannot be parse from separate elements.

**Additional context**
Fixes issue #3142.

### CODE REVIEW

File: src/connectors/fluxfm.js [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
+Connector.playerSelector = '.hero-player__headline';
 
-Connector.artistSelector = '#track-artist';
-
-Connector.trackSelector = '#track-title';
-
-Connector.albumSelector = '#track-album';
-
-Connector.trackArtSelector = '#covercontainer img';
+Connector.getArtistTrack = () => {
```
Comment (inverse) [inline src/connectors/fluxfm.js:5]: Could be simplified to

```
Connector.artistTrackSelector = '.hero-player__title';
```

**Review by inverse — COMMENTED** [review]
Looks good aside the one comment 👍🏼

**Comment by fckawe** [comment]
Okay, I didn't notice that one in the documentation... :/ Thanks for the review!
I'm not sure how to change that now, since I'm not so familiar with the whole workflow. Do I have to do something at all?

**Comment by fckawe** [comment]
Wait! I've made a mistake, sorry.
In my first version I took the artist - title element and separated it manually (via split). Then I've found the function to do the split artist and title.
The problem: The page is showing the title first and then the artist.
So, with that version artist and title are swapped! I didn't notice that because the music they played was quite unknown... :(
I guess I have to change it back to manually splitting...

**Comment by fckawe** [comment]
Commited two more changes:

1. Using different selectors, now. That way the connector not only works with the live radio player but also with the "channels" they provide on the website.
2. Disabling scrobbling if track title is "Livestream" and artist is "FluxFM", which sometimes happens when no music is played and no upcoming track is given.

I think it works quite well, now.

File: src/connectors/fluxfm.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+	const artistTrack = Util.getTextFromSelectors('.playbar__info__artist');
+	return Util.splitArtistTrack(artistTrack, null, { swap: true });
+};
 
 Connector.isPlaying = () => {
-	return $('#controls-play').hasClass('play');
+	return Util.getAttrFromSelectors('rs-play-button > rs-button > button', 'title') === 'Pausieren';
```
Comment (inverse) [inline src/connectors/fluxfm.js:11]:
```suggestion
	return Util.getAttrFromSelectors('.playbar rs-play-button > rs-button > button', 'title') === 'Pausieren';
```

**Review by inverse — CHANGES_REQUESTED** [review]
Minor change but looks a lot better 👍🏼

**Review by inverse — APPROVED** [review]
Modified with changes after testing locally 👍🏼

**Pull request merged**